packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ClaimModal/ClaimEthForm.tsx

```diff
@@ -98,15 +107,18 @@ export const ClaimEthForm = () => {
                 <Translation id="TR_STAKE_CLAIM_IN_NEXT_BLOCK" />
             </ClaimingPeriodWrapper>
 
-            <Button
-                type="submit"
-                isFullWidth
-                isDisabled={isDisabled}
-                isLoading={isComposing || isSubmitting}
-                onClick={handleSubmit(signTx)}
-            >
-                <Translation id="TR_STAKE_CLAIM" />
-            </Button>
+            <Tooltip content={claimingMessageContent}>
+                <Button
+                    type="submit"
+                    isFullWidth
+                    isDisabled={isDisabled || isClaimingDisabled}
```

Comment:
nitpick: I would expect `isClaimingDisabled` be part of `isDisabled`, but feel free to ignore